migration/migration.go

```diff
@@ -394,6 +394,7 @@ func GetMigrations() Migrations {
 	// Version 87
 	m = append(m, steps{ExecuteSQLFile("087-space-templates.sql",
 		spacetemplate.SystemLegacyTemplateID.String(),
+		spacetemplate.SystemBaseTemplateID.String(),
```

Comment:
Are you supposed to change a migration step that was already processed ?

Reply:
@xcoulon I didn't include the error from the migration before but please check the description of this PR now. The migration failed and needed to be fixed because otherwise any followup migrations wouldn't be reached.

Reply:
ah ok, now I understand why you fixed an older step. But it's not clear to me: did it fail in production as well ? i.e, the `version` was blocked at `86` because step `87` failed (which you're fixing now) ?

Reply:
If migration to version X has failed such that current prod-preview DB version is at X-1 , the migration step X can be changed.

Reply:
> ah ok, now I understand why you fixed an older step. But it's not clear to me: did it fail in production as well ? i.e, the version was blocked at 86 because step 87 failed (which you're fixing now) ?

@xcoulon currently we're only talking about prod-preview. And whenever a migration fails we won't increment the version.

Reply:
ok, good to know it's only on prod-preview. @sbose78 yes, this is what I wanted to have confirmed, indeed.